config.schema.json

```
@@ -65,6 +65,10 @@
               {
                 "title": "Weewx",
                 "const": "weewx"
+              },
+              {
+                "title": "Tempest Weather Station",
+                "const": "smartweather"
```

Nitpick:
```suggestion
                "title": "Tempest Weather Station",
                "const": "tempest"
```

IMHO the name should be `tempest` or `weatherflowtempest` or something along those lines, not `smartweather`, because https://weatherflow.github.io/SmartWeather/api/udp/v119/ is dead; https://weatherflow.github.io/Tempest/api/udp/v171/ is the up-to-date URL. So it seems the Weatherflow company has renamed it.